mktxp/collector/netwatch_collector.py

```diff
@@ -25,11 +26,61 @@ def collect(router_entry):
+            yield BaseCollector.gauge_collector('netwatch_icmp_rtt_min_ms', 'Netwatch ICMP Round Trip Min', [record for record in netwatch_records if record.get("type", None) == "icmp"], 'rtt_min', ['name', 'type'])
+            yield BaseCollector.gauge_collector('netwatch_icmp_rtt_max_ms', 'Netwatch ICMP Round Trip Max', [record for record in netwatch_records if record.get("type", None) == "icmp"], 'rtt_max', ['name', 'type'])
+            yield BaseCollector.gauge_collector('netwatch_icmp_rtt_jitter_ms', 'Netwatch ICMP Round Trip Jitter', [record for record in netwatch_records if record.get("type", None) == "icmp"], 'rtt_jitter', ['name', 'type'])
+            yield BaseCollector.gauge_collector('netwatch_icmp_rtt_stdev_ms', 'Netwatch ICMP Round Trip Stdev', [record for record in netwatch_records if record.get("type", None) == "icmp"], 'rtt_stdev', ['name', 'type'])
+
+            # TCP specific
+            yield BaseCollector.gauge_collector('netwatch_tcp_connect_time_ms', 'Netwatch TCP Connect Time', [record for record in netwatch_records if record.get("type", None) == "tcp-conn"], 'tcp_connect_time', ['name', 'type'])
+
+            # HTTP(s) specific
+            yield BaseCollector.gauge_collector('netwatch_http_status_code', 'Netwatch HTTP status code', [record for record in netwatch_records if record.get("type", None) in ["http-get", "https-get"]], 'http_status_code', ['name', 'type'])
+            yield BaseCollector.gauge_collector('netwatch_http_resp_time', 'Netwatch HTTP status code', [record for record in netwatch_records if record.get("type", None) in ["http-get", "https-get"]], 'http_resp_time', ['name', 'type'])
+            yield BaseCollector.gauge_collector('netwatch_tcp_connect_time_ms', 'Netwatch TCP Connect Time', [record for record in netwatch_records if record.get("type", None) in ["http-get", "https-get"]], 'tcp_connect_time', ['name', 'type'])
+
+    # Helpers
+    @staticmethod
+    def _translated_values(label, value):
+        try:
+            return {
+                'rtt_avg': lambda value: NetwatchCollector.parse_timedelta(value).microseconds/1000,
+                'rtt_min': lambda value: NetwatchCollector.parse_timedelta(value).microseconds/1000,
+                'rtt_max': lambda value: NetwatchCollector.parse_timedelta(value).microseconds/1000,
+                'rtt_jitter': lambda value: NetwatchCollector.parse_timedelta(value).microseconds/1000,
+                'rtt_stdev': lambda value: NetwatchCollector.parse_timedelta(value).microseconds/1000,
+                'tcp_connect_time': lambda value: NetwatchCollector.parse_timedelta(value).microseconds/1000,
+                'http_resp_time': lambda value: NetwatchCollector.parse_timedelta(value).microseconds/1000,
+            }[label](value)
+        except KeyError:
+            return value
+
+    @staticmethod
+    def parse_timedelta(time):
```

Comment:
The one in `BaseOutputProcessor` breaks because the `m` for minutes matches the `ms` in the value